src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,7 +153,7 @@ public void setBuyer(Buyer target, Buyer editedBuyer) {
      */
     @Override
     public ObservableList<Property> getFilteredPropertyList() {
-        return filteredProperties;
+        return filteredAndSortedProperties;
```

Comment:
This is pretty weird isn't it? Lets discuss the functionality with the group first before we see if we can improve the implementation

Reply:
SortedProperties is the definitive view as it is the final part of the pipeline.
The list pipeline is like this:
ObservableList -> FilterList -> SortedList
Hence, SortedList is the final step of the process and is what will be displayed.